addons/dragging/functions/fnc_handleScrollWheel.sqf

```diff
@@ -17,35 +17,47 @@
 
 params ["_scrollAmount"];
 
-private _unit = ACE_player;
-
 // EH is always assigned. Exit and don't overwrite input if not carrying
-if !(_unit getVariable [QGVAR(isCarrying), false]) exitWith {false};
+if !(ace_player getVariable [QGVAR(isCarrying), false]) exitWith {false};
 
-// move carried item 15 cm per scroll interval
-_scrollAmount = _scrollAmount * 0.15;
+private _unit = ace_player;
```

Comment:
ACE_player.

Why move it down here though?

Reply:
I don't want to create a variable each time the player uses it's scrollwheel without carrying an item.

Reply:
You're not going to save a single frame by this in 1000 hours of playing A3 while continiuously turning the mouse wheel.
I don't even think your mouse will survive before you saved one second.